webpack/assets/javascripts/react_app/components/bookmarks/__snapshots__/bookmarks.test.js.snap

```diff
@@ -1804,33 +1792,33 @@ exports[`bookmarks should show no bookmarks if server did not respond with any 1
                         >
                           <SafeAnchor
                             componentClass="a"
-                            id="bookmarkDocumentation"
                             onClick={[Function]}
                             role="menuitem"
                             tabIndex="-1"
                           >
                             <a
                               href="#"
-                              id="bookmarkDocumentation"
                               onClick={[Function]}
                               onKeyDown={[Function]}
                               role="menuitem"
                               tabIndex="-1"
                             >
-                              <Component
-                                className="icon-black"
-                                type="question-sign"
-                              >
-                                <span
-                                  className="glyphicon glyphicon-question-sign icon-black"
-                                />
-                              </Component>
-                               Documentation
+                              <DocumentLinkContent>
+                                <Component
+                                  className="icon-black"
+                                  type="question-sign"
+                                >
+                                  <span
+                                    className="glyphicon glyphicon-question-sign icon-black"
+                                  />
+                                </Component>
+                                 Documentation
+                              </DocumentLinkContent>
                             </a>
                           </SafeAnchor>
                         </li>
                       </MenuItem>
-                    </Component>
+                    </DocumentationLink>
                     <MenuItem
                       bsClass="dropdown"
                       disabled={false}
```

Comment:
nitpick: this is too large snapshot, is there a way to reduce its size? (divide it ?)

Reply:
Will have a solution for that hopefully in 2 hours :+1: